readthedocs/builds/models.py

```diff
@@ -132,6 +133,37 @@ def __str__(self):
             ),
         )
 
+    @property
+    def ref(self):
+        if self.slug == STABLE:
+            stable = determine_stable_version(self.project.versions.all())
+            if stable:
+                return stable.slug
+
+    @property
+    def vcs_url(self):
```

Comment:
I'm surprised we don't already have this logic somewhere.

Reply:
If we do have it, I didn't find it :/

Reply:
We assembled the url here https://github.com/rtfd/sphinx_rtd_theme/blob/a49a812c8821123091166fae1897d702cdc2d627/sphinx_rtd_theme/breadcrumbs.html#L45 but we get some metadata from here https://github.com/rtfd/readthedocs.org/blob/humitos/api/v3-implementation/readthedocs/doc_builder/backends/sphinx.py#L88-L104